doc/man7/provider-base.pod

```diff
@@ -372,7 +372,8 @@ additional ones.
 
```

Comment:
Introductory text above talks about "key exchange". Should it talk about "key exchange or encapsulation"?

Reply:
fixed in upcoming commit